addOns/reports/src/main/javahelp/org/zaproxy/addon/reports/resources/help/contents/report-traditional-json.html

```diff
@@ -26,7 +26,7 @@ <H3>Sample</H3>
                     "name": "Cross Site Scripting (Reflected)",
                     "riskcode": "3",
                     "confidence": "2",
-                    "riskdesc": "High (Medium)",
+                    "riskdesc": "High (Medium)", // Risk Level (Confidence level)
```

Comment:
I'd suggest documenting this between the `H3` and `pre` tags. Maybe something more like:

```html
<H4>riskdesc</H4>
riskdesc - Is a combination identifier, showing Risk followed by Confidence (in brackets). For example <code>High (Medium)</code> would indicate a High risk issue with Medium confidence.
```

Reply:
@thc202 @psiinon @ricekot thoughts?

Reply:
That seems a better place.